README.rst

```diff
@@ -65,13 +58,13 @@ for more information about service accounts. Use the `Google Cloud IAM web conso
 Development
 -----------
 To develop on the CLI, first run ``pip install -r requirements-dev.txt``. You can install your locally modified copy of 
-the hca package by running ``make install`` in the repository root directory.
+the ``dbio`` package by running ``make install`` in the repository root directory.
 
-To use the command line interface with a local or test DSS, first run ``hca`` (or ``scripts/hca`` if you want to use the
-package in-place from the repository root directory). This will create the file ``~/.config/hca/config.json``, which you
+To use the command line interface with a local or test DSS, first run ``dbio`` (or ``scripts/dbio`` if you want to use the
+package in-place from the repository root directory). This will create the file ``~/.config/dbio/config.json``, which you
```

Comment:
for the config file to be changed you have to modify https://github.com/DataBiosphere/data-store-cli/blob/c1e381f917c9b607a64b3243434470d578242559/hca/config.py#L12 the `name` here to dbio

Reply:
Will make this change in a later PR... Thanks for pointing out the location in the source code!

Reply:
Addressed in #18